initfiles/bin/init_thorslave

```diff
@@ -33,11 +33,23 @@ slavename=thorslave_${hpcc_compname}
 
 stop_slaves()
 {
-    killall -0 $slavename > /dev/null 2>&1
-    if [[ $? -eq 0 ]];then
-        log "killing slaves"
+    local timer=15
+    local isAlive=0
+
+    log "Attempting to kill $slavename with SIGTERM"
+    killall -15 $slavename > /dev/null 2>&1
```

Comment:
is 15 sigterm? Would it be better to specify the signal by name (I believe killall supports that)